environment.yml

```diff
@@ -0,0 +1,28 @@
+name: pyviz-topics
+channels:
+  - pyviz/label/dev
+dependencies:
+  - python=3.6
+  - notebook ==5.7.8
+  - ipykernel ==5.1.0
+  - nomkl
+  - bokeh ==1.1.0
+  - cartopy ==0.17.0
+  - colorcet ==2.0.1
+  - dask ==1.2.2
+  - datashader ==0.7.0
+  - fastparquet ==0.3.0
+  - geoviews ==1.6.2
+  - holoviews ==1.12.3
+  - lxml
+  - matplotlib
```

Comment:
Shouldn't mpl be pinned?  It does get some big updates...

Reply:
Yes. Everything will be pinned. Just trying to get what's needed.